scipy/sparse/linalg/eigen/lobpcg/tests/test_lobpcg.py

```diff
@@ -216,14 +217,54 @@ def test_hermitian():
         else:
             B = np.random.rand(size, size) + 1.j * np.random.rand(size, size)
             B = 10 * np.eye(size) + B.dot(B.T.conj())
-            w, v = lobpcg(H, X, B, maxiter=5000)
+            w, v = lobpcg(H, X, B, maxiter=5000, largest=False)
             w0, v0 = eigh(H, B)
 
         for wx, vx in zip(w, v.T):
             # Check eigenvector
-            assert_allclose(np.linalg.norm(H.dot(vx) - B.dot(vx) * wx) / np.linalg.norm(H.dot(vx)),
+            assert_allclose(np.linalg.norm(H.dot(vx) - B.dot(vx) * wx)
+                            / np.linalg.norm(H.dot(vx)),
                             0, atol=5e-4, rtol=0)
 
             # Compare eigenvalues
             j = np.argmin(abs(w0 - wx))
             assert_allclose(wx, w0[j], rtol=1e-4)
+
+def test_eigs_consistency():
+    n = 20
+    vals = np.arange(1, n+1, dtype=np.float64)
+    A = spdiags(vals, 0, n, n)
+    X = np.random.rand(n, 2)
+    lvals20, lvecs20 = lobpcg(A, X, largest=True, maxiter=100)
+    vals20, vecs20 = eigs(A, k=2)
+
+    _check_eigen(A, lvals20, lvecs20, atol=1e-3, rtol=0)
+    assert_allclose(vals20, lvals20, atol=1e-14)
+
+    # This tests the alternative branch using eigh().
+    n = 5
+    vals = np.arange(1, n+1, dtype=np.float64)
+    A = spdiags(vals, 0, n, n)
+    X = np.random.rand(n, 2)
+    lvals5, lvecs5 = lobpcg(A, X, largest=True, maxiter=100)
+    vals5, vecs5 = eigs(A, k=2)
+
+    _check_eigen(A, lvals5, lvecs5)
+    assert_allclose(vals5, lvals5, atol=1e-14)
+
+def test_verbosity():
+    """Check that nonzero verbosity level code runs.
+    """
+    A, B = ElasticRod(100)
+
+    n = A.shape[0]
+    m = 20
+
+    np.random.seed(0)
+    V = rand(n,m)
+    X = linalg.orth(V)
+
+    eigs,vecs = lobpcg(A, X, B=B, tol=1e-5, maxiter=30, largest=False,
+                       verbosityLevel=1)
+    eigs,vecs = lobpcg(A, X, B=B, tol=1e-5, maxiter=30, largest=False,
+                       verbosityLevel=10)
```

Comment:
The lines are identical apart from the `verbosityLevel`. I'm tempted to suggest parametrization here too, but I know some people don't like that unless there are > 2 cases.